code/game/machinery/painter_vr.dm

```diff
@@ -17,7 +17,8 @@
 	var/list/allowed_types = list(
 			/obj/item/clothing,
 			/obj/item/weapon/storage/backpack,
-			/obj/item/weapon/storage/belt
+			/obj/item/weapon/storage/belt,
+			/obj/item/device/radio/headset
```

Comment:
How is this related?

Reply:
It's not, I actually have no clue how that ended up being changed.